libyul/optimiser/UnusedAssignEliminator.cpp

```diff
@@ -24,48 +24,52 @@
 
 #include <libyul/optimiser/Semantics.h>
 #include <libyul/optimiser/OptimizerUtilities.h>
+#include <libyul/ControlFlowSideEffectsCollector.h>
 #include <libyul/AST.h>
+#include <libyul/AsmPrinter.h>
 
 #include <libsolutil/CommonData.h>
 
 #include <range/v3/action/remove_if.hpp>
 
+#include <iostream>
+
 using namespace std;
 using namespace solidity;
 using namespace solidity::yul;
 
+// TODO this component does not handle reverting function calls specially. Is that OK?
+// We should set m_activeStores to empty set for a reverting function call, like wo do with `leave`.
+
 void UnusedAssignEliminator::run(OptimiserStepContext& _context, Block& _ast)
 {
-	UnusedAssignEliminator rae{_context.dialect};
+	UnusedAssignEliminator rae{
```

Comment:
What do acronyms such as `rae`, `rse`, etc. mean?

Reply:
rae is redundant assign eliminator, it was renamed at some point and I guess I forgot this local variable.